salt/utils/yamldumper.py

```diff
@@ -108,12 +126,20 @@ def dump(data, stream=None, **kwargs):
     """
     .. versionadded:: 2018.3.0
 
+    .. versionchanged:: 3006.0
+
+        The default ``Dumper`` class is now ``OrderedDumper`` instead of
+        ``yaml.Dumper``.
+
     Helper that wraps yaml.dump and ensures that we encode unicode strings
     unless explicitly told not to.
     """
-    if "allow_unicode" not in kwargs:
-        kwargs["allow_unicode"] = True
-    kwargs.setdefault("default_flow_style", None)
+    kwargs = {
+        "allow_unicode": True,
```

Comment:
```suggestion
        "allow_unicode": kwargs.get("allow_unicode", True),
```

Just so this doesn't become a breaking change?

Reply:
No need.  If `"allow_unicode"` is already in `kwargs` then the `**kwargs` line below will cause its value to replace this value.